notebooks/tr-TR/fine_tuning_code_llm_on_single_gpu.ipynb

```diff
@@ -0,0 +1,1129 @@
+{
```

Comment:
<blockquote>Inference için modeli çağırdığımız hücreye geri dönerseniz ve fine-tune edilmiş ağırlıkları birleştirdiğimiz satırları yorum satırına eklerseniz, orijinal modelin tam olarak aynı önek için ne üreteceğini görebilirsiniz</blockquote><p> </p><p>Modeli Inference için başlattığımız notebook hücresine geri dönersek ve Fine-Tune edilmiş Weights ile birleştirdiğimiz satırları yoruma alırsanız, aynı önek için orijinal modelin ne üreteceğini görebilirsiniz.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>örnek* olacak aslında. bir de fine-tune yazsak olacak capitalize etmemize gerek yok</p>